src/vulkan/pipeline.h

```diff
@@ -105,6 +111,7 @@ class Pipeline {
     std::vector<std::unique_ptr<Descriptor>> descriptors_;
   };
 
+  VkPushConstantRange GetPushConstantRange();
```

Comment:
**METHOD COMMENTS PLEASE**

Reply:
Such as, are any stage bits set?
What's the offset?

Reply:
done